RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -36,7 +37,7 @@
                                  particleFlowEGammaFinal*
                                  pfParticleSelectionSequence )
 
-particleFlowLinks = cms.Sequence( particleFlow*particleFlowPtrs*particleBasedIsolationSequence)
+particleFlowLinks = cms.Sequence( particleFlow*particleFlowPtrs*chargedHadronIsolation*particleBasedIsolationSequence)
```

Comment:
OK, I guess a more strong reason to keep this one separate is if it stays requiring the PFBlock information, in which case it's logically less appropriate to couple it with the PFCand-based one.

Maybe to better reflect what this one is doing, change the name to chargedHadronTrackIsolation or chargedHadronPFTrackIsolation?